.travis.yml

```diff
@@ -32,10 +32,10 @@ matrix:
           env: wxGTK_VERSION=3 wxTOOLSET=cmake wxCMAKE_GENERATOR="Unix Makefiles"
           name: wxGTK 3 CMake Ubuntu 18.04
         - os: osx
-          osx_image: xcode6.4
+          osx_image: xcode7.3
           compiler: clang
           env: wxCONFIGURE_FLAGS="--enable-cxx11" wxMAKEFILE_FLAGS="CXXFLAGS=-std=c++11" wxSKIP_SAMPLES=1
```

Comment:
Not directly related to your changes, I wonder if we should remove C++11 options from this one, just to keep testing C++98 as long as we claim to continue supporting it (all the other Travis CI builds use C++11 or even C++17). Or maybe Xcode 7.3 compiler already uses C++11 by default anyhow? I don't know where can I find this information about Apple clang...

Reply:
I think C++98 is default behavior, I have found two tables, so to me it looks like all C++11 and C++14 features should be working

https://trac.macports.org/wiki/XcodeVersionInfo
https://clang.llvm.org/cxx_status.html

so if we have this Xcode version as a minimum, why would we have to support C++98 on macOS ?

Reply:
Actually I've just checked and the default is still C++98 even in Xcode 10.2 (which I currently have), strangely enough.

As for supporting C++98, I'm fine with not supporting it any longer under macOS, but then we need to enable C++11 in configure automatically and document that C++98 is not supported any more. It's also somewhat inconsistent to not support it under macOS while we still support it under Linux/other Unices, I'd rather drop it everywhere at once.

Reply:
I can see the point there, but I don't want to wait too long;-)
Is it on other platforms the same situation that all the compilers we support, also support c++11 ? so that we don't loose a platform if we decide this ?

Reply:
No, unfortunately it's still not the case. There are CentOS/RHEL 6 systems (which I have to target...) that don't have C++11 compiler out of the box and only 4.8 (which doesn't have great C++11 support) can be easily installed on them. Also some non-desktop platforms, but wx is probably not too relevant there (although sometimes it's still used). And, of course, there are still all the MSVS versions earlier than 2013, which was the first one to have support for a decent subset of C++11, that we still support. I'm afraid C++98 will still be supported in 3.2.0.

BTW, do we still need `-stdlib=libc++` with the (now) supported Xcode versions or can I remove the part adding it from configure?

Reply:
for Xcode itself we don't need it, once you are above or at macos-min 10.9, I'd assume command line builds do likewise, I'll check

Reply:
> I'm afraid C++98 will still be supported in 3.2.0.

So then I'd really like to have at least in the macOS code, as we don't loose any systems.